web_responsive/static/src/js/web_responsive.js

```diff
@@ -599,7 +599,7 @@ odoo.define("web_responsive", function (require) {
                         let values = [];
                         if (filter.groups) {
                             values = [
-                                ...filter.groups.values().map((g) => g.values),
+                                ...[...filter.groups.values()].map((g) => g.values),
```

Comment:
We could make it a little more readable by using...
`values = Array.from(filter.groups.values(), item => item.values)`

How do you see it? Anyway, thanks for the patch.

Reply:
Thanks, it's clearer.
Tested this variant on runbot. It works.